certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1,1411 +1,1422 @@
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status"
+                },
+                "bookName": {
+                  "type": "string",
+                  "description": "bookName"
+                },
+                "bookID": {
+                  "type": "string",
+                  "description": "bookID"
+                }
+              }
+            }
+          }
+        },
+        "summary": "BookBuilder - Create Book from Template",
+        "description": "This endpoint retrieves book status, book name, and book ID for ticker requested in JSON format. This end-point accepts ticker and template_id as inputs. If the template_id input is not used, a book will be created with FactSet's default template.",
+        "operationId": "create-book-from-template",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "ticker": {
+                  "type": "string",
+                  "description": "ticker"
+                },
+                "template_id": {
+                  "type": "string",
+                  "description": "template_id"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/book-builder-api/v1/download-api-book/{book_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        },
+        "summary": "BookBuilder - Get PDF",
+        "description": "This endpoint will return the PDF output given a book_id.",
+        "operationId": "get-pdf",
+        "parameters": [
+          {
+            "name": "book_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Book ID",
+            "description": "ID of the book to download"
+          }
+        ]
+      }
+    },
+    "/content/symbology/v3/identifier-resolution": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Symbology API",
+        "description": "Translate Entity ID and market security symbols into FactSet Permanent Security Identifiers.",
+        "operationId": "get-symbology",
+        "parameters": [
+          {
+            "name": "ids",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "inputSymbolType",
+            "in": "query",
+            "required": false,
+            "type": "string"
```

Comment:
@KyleVErwin The 'x-ms-summary' property is required. The 'description' property is required. Please add both to the parameter.